### ISSUE TITLE
Research barcode cutting machine

### ISSUE THREAD

**Issue by PipBrewer** [NHMDenmark/Pinned-Insects-workstation#19]

**Comment by PipBrewer** [comment]
Transferred from [Solicit Feedback on paper cutting machine and do research into alternatives#237](https://github.com/NHMDenmark/Mass-Digitizer/issues/237) to put in correct repo

**Comment by chelseagraham** [comment]
Currently have had success with cutting with paper cutter

**Comment by Gomismis** [comment]
If cutting the labels manually is too time consuming I suggest that we buy a CRICUT machine:

https://www.proshop.dk/Labelprinter/Cricut-Maker-3/3183865?utm_source=google&utm_medium=cpc&utm_campaign=searchengine&gad_source=1&gclid=CjwKCAiA5L2tBhBTEiwAdSxJX-IRuJND3CFlpcHDCIWo0OW4pZXWxh-taF_OkWIcChHRmOh5-ESOxxoCW5kQAvD_BwE

It's very precise and can cut most materials.

**Comment by chelseagraham** [comment]
Is there a possibility that such a machine can pre-prick the pinhole? This would facilitate pinning the labels on very thin pins.

**Comment by RebekkaML** [comment]
> Is there a possibility that such a machine can pre-prick the pinhole? This would facilitate pinning the labels on very thin pins.

I think this would only make sense if it can make a very tiny pinhole (as small or slightly smaller than the thinnest pin) because there is a lot of variation in pin thickness and if the pinhole is too big the label doesn't stay on.

**Comment by chelseagraham** [comment]
I'm pretty sure @Gomismis tested this machine or something similar this spring / early summer. If so, would you please update this ticket with your thoughts, Matilde?

**Comment by Gomismis** [comment]
I don't think it can make holes that small, unfortunately. I’ve been doing some testing, trying to make it cut out rows of labels. It turned out to be much more difficult than I first anticipated, so I never managed to adjust it for a perfect cut. I might be able to do so, but it would take some days of work/testing.

**Comment by chelseagraham** [comment]
> I don't think it can make holes that small, unfortunately. I’ve been doing some testing, trying to make it cut out rows of labels. It turned out to be much more difficult than I first anticipated, so I never managed to adjust it for a perfect cut. I might be able to do so, but it would take some days of work/testing.

Maybe we should discuss this at the next Digitizer meeting. (@RebekkaML , would you please add it to the agenda?) I am thinking the juice is not worth the squeeze with this one. I don't think it is worth investing a lot of time in this machine at this stage. The paper cutter is working alright. It seems unlikely that an automated machine will cut precisely enough and be able to make holes small enough for those pesky thin pins.

**Comment by chelseagraham** [comment]
We discussed this in the Digitzer Meeting and decided to shelf this.
